pallets/nfts/src/lib.rs

```diff
@@ -199,8 +210,29 @@ pub mod pallet {
     /// Data related to NFTs.
     #[pallet::storage]
     #[pallet::getter(fn data)]
-    pub type Data<T: Config> =
-        StorageMap<_, Blake2_128Concat, T::NFTId, NFTData<T::AccountId, T::NFTDetails>, ValueQuery>;
+    pub type Data<T: Config> = StorageMap<
+        _,
+        Blake2_128Concat,
+        T::NFTId,
+        NFTData<T::AccountId, T::NFTDetails, T::NFTSeriesId>,
+        ValueQuery,
+    >;
+
+    /// The number of NFT Series managed by this pallet
+    #[pallet::storage]
+    #[pallet::getter(fn total_series)]
+    pub type TotalSeries<T: Config> = StorageValue<_, T::NFTSeriesId, ValueQuery>;
+
+    /// Data related to NFT Series.
+    #[pallet::storage]
+    #[pallet::getter(fn series)]
+    pub type Series<T: Config> = StorageMap<
+        _,
+        Blake2_128Concat,
+        T::NFTSeriesId,
+        (T::AccountId, sp_std::vec::Vec<T::NFTId>),
```

Comment:
This could make sense to define a struct `NFTSerieDetails` to hold the owner information and associated nfts. Btw you can probably import `sp_std::vec::Vec` instead of using it directly

Reply:
Done :)